pdal/Stage.hpp

```diff
@@ -387,6 +387,8 @@ class PDAL_EXPORT Stage
     void setupLog();
     void handleOptions();
     void countElements(const PointViewSet& views);
+    // set subclass-specific options after they've been processed
+    virtual void assignParsedOptions();
```

Comment:
A consequence of adding this virtual is previous plugins will need to be recompiled. Just putting this in so I have a reminder when I make release notes.

Reply:
If this is at the end of the list of virtual functions, it should probably be fine. We can try to think of another way to deal with this, too.

Reply:
I don't think it is too much burden. We just need to highlight it when we release.